esmvaltool/diag_scripts/weighting/climwip.py

```diff
@@ -0,0 +1,438 @@
+"""Implementation of the climwip weighting scheme.
+
+Lukas Brunner et al. section 2.4
+https://iopscience.iop.org/article/10.1088/1748-9326/ab492f
+"""
+from collections import defaultdict
+import os
+import logging
+from datetime import datetime
+
+import yaml
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import xarray as xr
+from scipy.spatial.distance import pdist, squareform
+
+from esmvaltool.diag_scripts.shared import (ProvenanceLogger,
+                                            get_diagnostic_filename,
+                                            get_plot_filename, run_diagnostic,
+                                            select_metadata)
+
+logger = logging.getLogger(os.path.basename(__file__))
+
+
+def get_provenance_record(caption: str, ancestors: list):
+    """Create a provenance record describing the diagnostic data and plots."""
+    record = {
+        'caption': caption,
+        'domains': ['reg'],
+        'authors': [
+            'kalverla_peter',
+            'smeets_stef',
+            'brunner_lukas',
+            'camphuijsen_jaro',
+        ],
+        'references': [
+            'acknow_project',
```

Comment:
So we avoid 'acknow_project' altogether?

Reply:
Yes, that would be good, I think it should probably be completely removed, because it is not citation information. Apparently it was introduced for cases where no scientific publication was available.